README.md

```diff
@@ -1,86 +1,67 @@
-When you start the Ceramic daemon you will get an output that looks something like this:
+Create an issue using the *Peerlist Request* template filling in each section with the neccessary information. 
 
-```sh
-$ ceramic daemon
-# ...
-Swarm listening on /ip4/127.0.0.1/tcp/4002/p2p/QmNZfs1Vevknvmykz3fWKTjhmEpckabhd2JyEGJuymZFsC
-Swarm listening on /ip4/192.168.188.22/tcp/4002/p2p/QmNZfs1Vevknvmykz3fWKTjhmEpckabhd2JyEGJuymZFsC
-Swarm listening on /ip4/10.70.198.64/tcp/4002/p2p/QmNZfs1Vevknvmykz3fWKTjhmEpckabhd2JyEGJuymZFsC
-Swarm listening on /ip4/127.0.0.1/tcp/4003/ws/p2p/QmNZfs1Vevknvmykz3fWKTjhmEpckabhd2JyEGJuymZFsC
-```
-
-This is your IPFS multiaddress. It can also be viewed in the `config` file that the IPFS daemon creates which is located by default in a directory called `ipfs`. This directory gets created wherever IPFS runs. Of you are running IPFS out-of-process, you will see a similar log output on startup and you can simply get the multiaddress from there.
-
-It is recommended that you use a websocket multiaddress and configure SSL (e.g. `.../<port>/wss/p2p/<peerId>`). Before adding your multiaddress to the peerlist, be sure to replace the local IP addresses (e.g. `127.0.0.1`) with your external IP address or DNS name. Also ensure that the websocket port (`4003` in the example above) is open to internet connections. For more information see [multiaddress docs](https://github.com/multiformats/multiaddr).
+Once we receive your *Peerlist Request* issue we will review it and ask for more information if needed. When the *Peerlist Request* gets accepted we will add your IP address to our allow list. The *Peerlist Request* issue with then be closed and your node will be connected to the Ceramic network and the [Ceramic Anchor Service](https://github.com/ceramicnetwork/ceramic-anchor-service). You can create a *Peerlist Request* issue for the following networks: 
 
-### Connectivity test
-When a pull request is submitted, it triggers a connectivity test to make sure your node is successfully connected to the network. If this fails, the 3Box Labs team will reach out to you to triage the issue. Make sure there are no firewalls blocking your instance and that your port is properly exposed.
+- **`mainnet`** 
+- **`testnet-clay`** 
+- **`dev-unstable`** 
 
-Once you are on the peerlist, you should monitor your IPFS node and alert our team on Discord in the case of any planned or unexpected downtime. Please make your best effort to come back online within 24 hours. If we can not connect to your IPFS node for over 24 hours, we will remove it from the peerlist and you can resubmit your multiaddress in a new PR once your node becomes stable again. If the connectivity test in your PR to the peerlist fails and it is due to a node other than your own, we will update the peerlist and re-run the tests for you.
+It is recommended to first run your node on the Clay testnet to ensure it is set up correctly before running mainnet.
 
 ### Readiness checklist
```

Comment:
an interesting side-effect of removing the peerlist is that now we have no way for node operators to sanity check that their nodes are actually accessible to the network.  I suspect we'll see a rise in people running nodes, thinking they are working fine, but actually all their anchors are failing and none of their streams are available to other nodes on the network.  I wonder if we should think about doing something to make misconfigurations more obvious?  @oed

Reply:
Maybe add something to the Ceramic node where; if > N anchor requests fail warn in the console?

Reply:
We already log a warning on every failed anchor.  I'm just not convinced that anyone notices or cares given that their streams stay accessible and usable (usually).

I filed https://linear.app/3boxlabs/issue/NET-1305/use-cas-to-perform-a-network-availability-check-at-node-startup to address this concern more directly.